ServerCore/Pages/Teams/Create.cshtml

```diff
@@ -34,13 +34,12 @@ else
                 <input asp-for="Team.Name" class="form-control" />
                 <span asp-validation-for="Team.Name" class="text-danger"></span>
             </div>
-            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt)
-    <div class="form-group">
+            <!-- TODO: Conditionally show/hide based on event property (not needed for PuzzleHunt) -->
```

Comment:
Should this be removed?

Reply:
I'm not sure what the extra (unconditional) room label is and I think that that's what this applies to, so I want to leave it for the moment.